apps/epic-react/src/templates/article-template.tsx

```diff
@@ -83,8 +86,9 @@ const ArticleTemplate: React.FC<ArticleTemplateProps> = ({
   return (
     <Layout
       meta={{
-        title,
-        description: pageDescription,
+        title: truncate(title, {length: 75}),
+        description: truncate(pageDescription, {length: 155}),
+        url,
```

Comment:
added some seo and parameters around that. Titles shouldn't be above 75 char and descriptions 155. We need to pass url so the canonical url is set